xbmc/settings/Settings.cpp

```diff
@@ -456,6 +456,9 @@ bool CSettings::InitializeDefinitions()
 #elif defined(TARGET_ANDROID)
   if (CFile::Exists(SETTINGS_XML_FOLDER "android.xml") && !Initialize(SETTINGS_XML_FOLDER "android.xml"))
     CLog::Log(LOGFATAL, "Unable to load android-specific settings definitions");
+#elif defined(HAS_LIBAMCODEC)
```

Comment:
What about amlogic on android?
Do you have a precise idea of what you want to put in there?